src/components/views/spaces/SpaceTreeLevel.tsx

```diff
@@ -133,8 +133,8 @@ export const SpaceButton = forwardRef<HTMLElement, IButtonProps>(
         }
 
         const viewSpaceHome = (): void =>
-            defaultDispatcher.dispatch({ action: Action.ViewRoom, room_id: space.roomId });
-        const activateSpace = (): void => SpaceStore.instance.setActiveSpace(spaceKey ?? space.roomId);
+            defaultDispatcher.dispatch({ action: Action.ViewRoom, room_id: space!.roomId });
+        const activateSpace = (): void => SpaceStore.instance.setActiveSpace(spaceKey ?? space!.roomId);
```

Comment:
In case of `viewSpaceHome` the assignment to `onClick` ensures that `space` is set. I've updated the other case to pass an empty string to `setActiveSpace` because it handles this internally.